Makefile

```diff
@@ -28,8 +28,8 @@ fmt:  ## Run goimports on all go files
 
 .PHONY: github-tag
 github-tag:  ## Create and push a tag with the current version
-	git tag -a ${SDK_VERSION} -m "Synse SDK version ${SDK_VERSION}"
-	git push -u origin ${SDK_VERSION}
+	git tag -a v${SDK_VERSION} -m "Synse SDK version ${SDK_VERSION}"
+	git push -u origin v${SDK_VERSION}
```

Comment:
Why the introduction of the `v` prefix though? We use non-prefixed semvers for github releases & image tags in all repos

Reply:
I agree, this seems like an odd stop to inject a `v` .. it is either part of SDK_VERSION or not.

Reply:
Because the go.mod files would end up looking like:

https://github.com/vapor-ware/synse-emulator-plugin/blob/develop/go.mod#L13
https://github.com/vapor-ware/synse-i2c-plugin/blob/master/go.mod#L19
https://github.com/vapor-ware/synse-modbus-ip-plugin/blob/master/go.mod#L17

I'll move the `v` prefix into the env var